roles/common/tasks/main.yml

```diff
@@ -34,6 +34,18 @@
   when: item.value.site_hosts | rejectattr('canonical', 'defined') | list | count
   tags: [letsencrypt, wordpress]
 
+- name: Import PHP version specific vars
+  include_vars: "{{ lookup('first_found', params) }}"
+  vars:
+    params:
+      files:
+        - '{{ php_version }}.yml'
+        - '7.4.yml'
+      paths:
+        - "{{ playbook_dir }}/roles/php/vars/"
```

Comment:
Added this as a further fallback/default in case people don't get the update to `group_vars/all/main.yml`.